src/telegram.ts

```diff
@@ -1109,6 +1116,63 @@ export class Telegram extends ApiClient {
     })
   }
 
+  /**
+   * Use this method to change the bot's menu button in a private chat, or the default menu button. Returns true on success.
+   * @param chatId Unique identifier for the target private chat. If not specified, default bot's menu button will be changed.
+   * @param menuButton An object for the new bot's menu button.
```

Comment:
```suggestion
   * @param menuButton An object for the bot's new menu button.
```
Or was this copied from the API? In that case, we should request the Telegram docs to be changed.

Reply:
Yep, Telegram docs needs to be fixed
https://core.telegram.org/bots/api#setchatmenubutton